sound/soc/sof/intel/hda.h

```diff
@@ -344,7 +344,13 @@
 
 /* Number of DAIs */
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
+
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_PROBES)
+#define SOF_SKL_NUM_DAIS		16
```

Comment:
same here, this should be dynamic.

Reply:
Same answer as here: https://github.com/thesofproject/linux/pull/1276/files#r336180833.